ytmusicapi/ytmusic.py

```diff
@@ -127,9 +132,9 @@ def setup(cls, filepath: str = None, headers_raw: str = None) -> Dict:
         :return: configuration headers string
         """
         return setup(filepath, headers_raw)
-    
+
     def __enter__(self):
         return self
-    
-    def __exit__(self, execType = None, execValue = None, trackback = None):
+
+    def __exit__(self, execType=None, execValue=None, trackback=None):
```

Comment:
`yapf` did this, I think, and I couldn't work out how to undo it

Reply:
It's fine, don't worry about it